ajax/libs/moonjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "moonjs",
+  "filename": "moon.min.js",
+  "version": "0.10.0",
+  "description": "A minimal, blazing fast UI library.",
+  "homepage": "http://moonjs.ga/",
+  "keywords": [
+    "moon",
+    "mvvm",
+    "user interface",
+    "UI"
+  ],
+  "author": "Kabir Shah",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/KingPixil/moon"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/KingPixil/moon.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  }
```

Comment:
in this case, npm has more release versions than GitHub. Let's use npm auto-update, thank you.
```js
"npmName": "moonjs",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ]
```
ref: https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md